uw-research-computing/gpu-jobs.md

```diff
@@ -17,82 +17,16 @@ For researchers who have problems that are well-suited to GPU
 processing, it is possible to run jobs that use GPUs in CHTC. Read on to
 determine:
 
-1.  [Examining CHTC's GPU Capacity](#a-examining-chtcs-gpu-capacity)
-2.  [Using the CHTC GPU Lab](#b-using-the-chtc-gpu-lab)
-3.  [Preparing Software Using GPUs](#c-preparing-software-using-gpus)
-4.  [GPU Capacity Beyond the CHTC GPU Lab](#d-gpu-capacity-beyond-the-chtc-gpu-lab)
+- [What GPUs are available in CHTC](#a-available-chtc-gpus)
```

Comment:
My personal preference is to match the text in the list and the sub-heading below, but I don't feel strongly about that.